app/views/my/page_layout.html.erb

```diff
@@ -88,30 +88,57 @@ See doc/COPYRIGHT.rdoc for more details.
   <li><%= link_to l(:button_back), {:action => 'page'}, :class => 'icon icon-cancel' %></li>
 <% end %>
 
+<% content_for :toolbar do %>
+    <li class="toolbar-item">
+      <%= styled_form_tag({:action => "add_block"}, :id => "block-form") do %>
+        <%= styled_select_tag 'block', "<option>--#{t(:button_add)}--</option>".html_safe + options_for_select(@block_options),
+              :id => "block-select",
+              class: '-small' %>
+      <% end %>
+    </li>
+    <li class="toolbar-item">
+      <%= link_to_remote l(:button_add),
```

Comment:
At first I was irritated by `link_to_remote` because I thought "Isn't this a local link?"

Then I searched and found this:
````
# remove once we no longer use the deprecated "link_to_remote", "remote_form_for" and alike methods
# replace those with :remote => true
gem 'prototype_legacy_helper', '0.0.0', :git => 'https://github.com/rails/prototype_legacy_helper.git'
````

I am asking because you are adding it here (wasn't there before)